cdisc_rules_engine/rules_engine.py

```diff
@@ -174,9 +176,13 @@ def validate_single_rule(
                 return [error_obj.to_representation()]
         except Exception as e:
             logger.trace(e, __name__)
-            logger.error(
-                f"Error occurred during validation. Error: {e}. Error message: {str(e)}"
-            )
+            if not (
```

Comment:
I don't think we need to conditionally log here

Reply:
The reason to do conditional log is to make sure that when validate-xml flag is set to false, then the error message is not logged,

Reply:
I think it's ok if it is logged though

Reply:
removed condition for logging in latest commit